src/main/java/com/google/api/gax/grpc/ServiceApiSettings.java

```diff
@@ -106,6 +115,16 @@ public ManagedChannel getChannel(Executor executor) throws IOException {
             .build();
         return channel;
       }
+
+      private String serviceHeader() {
+        // GAX version only works when the package is invoked as a jar.
+        String gaxVersion = ChannelProvider.class.getPackage().getImplementationVersion();
+        String generatorName = serviceGeneratorVersion.isEmpty() ?
+            DEFAULT_GENERATOR_NAME : serviceGeneratorName;
+        String generatorVersion = serviceGeneratorVersion.isEmpty() ?
+            DEFAULT_GENERATOR_VERSION : serviceGeneratorVersion;
+        return String.format("gax-%s/java/%s-%s", gaxVersion, generatorName, generatorVersion);
```

Comment:
I just remembered, I think we want to put the java runtime version in here, instead of simply 'java'.